### PR TITLE
Update README & add CONTRIBUTING

### PR DESCRIPTION
**Pull request by eliascodes**
### Changes
* Update README; reflects contents of London & Naz
* Add CONTRIBUTING; reflects contents of London

FYI: @saralk @RanaAlqrenawi @des-des

### CODE REVIEW

**Comment by eliascodes** [comment]
@iteles I believe they will be the same. Your idea is a good one, except that international doesn't have a contributing document yet, and the one in the master-reference is completely different 😄

**Comment by jsms90** [comment]
I think this completely depends on whether we're using a centralised model, where FCI decides how the campuses are run, or whether the members of the campus should decide for themselves. While it might be useful for International to have a blueprint, for setting up new campuses, I personally think that the members of the campus should be able to change their own processes immediately.

In which case, this approach of duplicating documents is fine, because it allows them to diverge as soon as a campus wants.

@iteles @eliascodes @bradreeder @des-des What do you think?

**Comment by iteles** [comment]
@eliascodes I'm up for getting this into `international` as a `contributing.md` file today and setting it up the way we mean to go on. 
Speaking to @rebeccaradding in 45 mins and we'll discuss/agree then and report back 👍

**Comment by iteles** [comment]
Note: My intention is not to hold up the merging in of this PR (I 'approved' on review), we can always amend later.

File: README.md [hunk 1]
```diff
@@ -1,3 +1,44 @@
 # Gaza Programme
 
-A repository for the coordination of activities in Gaza.
+The Gaza Programme repository is a place for discussions by the wider membership and for posting proposals for a vote at the London business meetings.
```
Comment (des-des) [inline README.md]: `London` -- `Gaza`

File: README.md [hunk 2]
```diff
@@ -1,3 +1,44 @@
...
+
+[Gaza Curriculum](https://github.com/foundersandcoders/gaza-curriculum): Curriculum information for the current cohort, decisions by Curriculum Commitee
+
+[Gaza Membership](https://github.com/foundersandcoders/gaza-membership): Membership records, Membership Committee decisions
+
+[Selection Committee](https://github.com/foundersandcoders/selection-committee): Joint committee for determining selection criteria and processes
+
+## Other Important Founders & Coders Repositories
+
+[Master-Reference](https://github.com/foundersandcoders/master-reference): Records and discussions about all aspects of common curriculum
```
Comment (des-des) [inline README.md]: Also contains the actual curriculum..

**Comment by rebeccaradding** [comment]
@eliascodes the FCI version has been merged: https://github.com/foundersandcoders/international/blob/master/consensus-decisions.md

Yes other docs are on their way as soon as I can get them drafted. 

In London and Nazareth I have opened PR's to link to this doc from the README rather than uploading a file. I have also opened PR's to delete their current CONTRIBUTING.md docs.

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+
+You have a question or idea and want to know what other members think about it. 
+
+Our community needs a democratic decision-making process that facilitates collection action.
+
+---
+
+### WHAT to do to prepare a proposal for vote:
+
+**NOTE:** This process applies to _both_ whole-group decisions _and_ decisions make within committees.
```
Comment (mantagen) [inline CONTRIBUTING.md]: ~~make~~ made

**Pull request merged**